### PR TITLE
Add Post IO meetup file

### PR DESCRIPTION
**Pull request by DarshilBhatt**
What does this PR add / remove or fix?

Add new post, Post IO Meetup & one image in the assets folder for the same post. 

Fix the same post, by splitting content into paragraphs.

Does this PR match the guidelines documented in the Contributing Guide?

Yes

### CODE REVIEW

**Comment by manan-jadhav** [comment]
Please read up the contribution guide first : https://github.com/gdgbaroda/gdgbaroda.github.io/blob/master/CONTRIBUTING.md#writing-pull-requests

**Comment by DarshilBhatt** [comment]
I've already read. Can you please tell me what's the issue exactly?

**Comment by knightkill** [comment]
@DarshilBhatt 
You have to comment the changes while creating "New Pull request" follow this format:

What does this PR add / remove or fix?

Your answer here

Does this PR match the guidelines documented in the Contributing Guide?

Yes/No

**Comment by DarshilBhatt** [comment]
Done

File: _posts/2018-07-24-post-io-meetup.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
... could miss keynote & goes to back stage, he/she can definitely interact with Sundar Pichai & other speakers. After keynote, sessions on different technical stuffs started in the [codelabs](https://events.google.com/io/schedule/?section=may-8). Codelabs are small buildings where sessions on different tech-stuffs take place. There were codelabs for Android, Firebase, Web , Google Accessibility, Android & IoT, Machine Learning, Flutter and many more. Attendees can attend codelabs depending on their specific field of interest or they can even explore different stuffs. After one hour of codelabs, there was [Developer Keynote](https://events.google.com/io/schedule/?section=may-8), where speakers explained innovations in different development frameworks used for developing different products. This keynote too was a great experience for the co-organizers, by which developers can code efficient products. Keynote continued for one hour & again sessions in the codelabs began. And that codelab sessions were in motion on the second & third day of IO, for the whole day, starting from morning to evening. According to co-organizers, codelab can be a best thing to explore & learn different stuffs.
```
Comment (manan-jadhav) [inline _posts/2018-07-24-post-io-meetup.md]: Please break this into two paragraphs. Read "Updating PRs" sections of the contributing guide to see how to make updates to a PR.

**Review by manan-jadhav — CHANGES_REQUESTED** [review]

**Comment by DarshilBhatt** [comment]
Requested changes made.

**Review by manan-jadhav — APPROVED** [review]

**Pull request merged**